homeassistant/components/unifi/unifi_client.py

```diff
@@ -12,11 +12,7 @@ def __init__(self, client, controller) -> None:
         super().__init__(client, controller)
 
         self._is_wired = client.mac not in controller.wireless_clients
-
-    @property
-    def client(self):
-        """Wrap item."""
-        return self._item
+        self.client = self._item
```

Comment:
There is an occurence in UniFiDeviceTracker as well that has a property for device

Reply:
I'll switch that one as well.  Since there are lots more clients than devices, I didn't notice in the profile.